Blockzilla/AboutViewController.swift

```diff
@@ -39,7 +48,17 @@ class AboutViewController: UIViewController, UITableViewDataSource, UITableViewD
     }
 
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = UITableViewCell()
+        var cell : UITableViewCell?
+        cell = tableView.dequeueReusableCell(withIdentifier: "cellID")
```

Comment:
Could this be merged with the line above? `var cell = tableView.dequeueReusableCell(withIdentifier: "cellID")`

Reply:
Fixed